puppetfactory/puppetfactory.gemspec

```diff
@@ -3,7 +3,7 @@ require 'date'
 
 Gem::Specification.new do |s|
   s.name              = "puppetfactory"
-  s.version           = '0.5.5'
+  s.version           = '0.5.6'
```

Comment:
Ah ok, sorry about that. First time doing this :( Would it be better to unwind everything and redo it?